pkg/kubectl/resource/builder.go

```diff
@@ -192,6 +203,7 @@ func (b *Builder) Unstructured() *Builder {
 		return b
 	}
 	b.mapper = b.unstructured
+	b.mapper.localFn = b.isLocal
```

Comment:
why does Unstructured affect isLocal? whether or not you make use of conversion or typed objects depends on the typer/convertor you give to the builder, not whether it has a server connection and can look up restmappings, right?

Reply:
> why does Unstructured affect isLocal? whether or not you make use of conversion or typed objects depends on the typer/convertor you give to the builder, not whether it has a server connection and can look up restmappings, right?

It's the point where the mapper is chosen and the old spot can logically wire this piece of information.  A few more pulls and the mapper won't be specified by callers and this will effectively drop out.